### DISCUSSION TITLE
TTS not working

### DISCUSSION THREAD

**Discussion by Uberhot** [open-webui/open-webui#3978] [General]
# Bug Report

## Description
Webapi TTS engine not working. no clue why
**Bug Summary:**
Webapi TTS engine not working. no clue why.
Previously, the models were not displayed at all. appeared now, but still doesn't work... i tried all models but still nothing

**Steps to Reproduce:**
dunno?

**Expected Behavior:**
I must hear the voice. but nothing happens

**Actual Behavior:**
clicking on tts button but nothing happens

- **Operating System:** 
- arch linux i3wm
- **Browser:** 
- Firefox latest

## Logs and Screenshots

**Screenshots :**
![Screenshot 2024-07-18 at 18-16-15 Greeting 👋 Open WebUI](https://github.com/user-attachments/assets/3f1d5770-6cb1-4bca-a3a1-79d88b191926)

**Comment by SD-fan** [comment]
TTS using openedai works just fine. Just follow the docs:
https://docs.openwebui.com/tutorial/openedai-speech-integration

**Reply by Uberhot** [reply]
thx

**Reply by johnyu0** [reply]
> TTS using openedai works just fine. Just follow the docs: https://docs.openwebui.com/tutorial/openedai-speech-integration

I have configured the openai api pointed to  openedai speech interface(http://192.168.1.240:8000/v1) and saved it. 
But when i click read cloud button,  I have seen the open-webui still call the audio api of the open-webui. The console log is:
index.ts:105
       POST http://192.168.1.240:3000/audio/api/v1/speech 500 (Internal Server Error)
       
I think the requests should be posted to http://192.168.1.240:8000/v1/XXXXXXX

Thank you

**Comment by johnyu0** [comment]
![image](https://github.com/user-attachments/assets/9804f381-aa23-422f-9e01-25e0ad1ab4b6)
This is my configuration, and host.docker.internal can be resolved as 192.168.1.240.
